rtos/source/TARGET_CORTEX/mbed_rtos_rtx.c

```diff
@@ -107,6 +153,10 @@ MBED_NORETURN void mbed_rtos_start()
     tfm_ns_lock_init();
 #endif // defined(FEATURE_TFM) && defined(FEATURE_NSPE)
 
+#if defined(TARGET_TFM_TWINCPU)
+    tfm_ns_multi_core_boot();
```

Comment:
I am not sure if `mbed_rtos_start()` is the correct place to hook in behavior that applies to all `TFM_TWINCPU` targets. Do we have a better place to hook? A worse place to hook would be in target specific code that would require copy-pasting for any new `TFM_TWINCPU` target; that'd be a bad experience for folks bringing new TF-M targets to Mbed OS.

@bulislaw Any recommendations? Is this really the best place?

Reply:
I'm not sure what you need/want to achieve and by when. Actually adding a comment for this function's intent could be good. Don't get me wrong I can read what it does, but I lack background to actually understand implications and the best place to do it or the requirements. Looking at the other changes it has dependency on RTOS - mutex, so that would require RTOS to actually run. The bootstrap process is described here https://github.com/ARMmbed/mbed-os/blob/master/rtos/source/TARGET_CORTEX/mbed_boot.h